client/state/data-layer/wpcom/sites/gutenberg/index.js

```diff
@@ -35,8 +36,8 @@ const dispatchSelectedEditorRequest = dispatchRequestEx( {
 	onError: noop,
 } );
 
-export const setType = action =>
-	http(
+export const setType = action => {
+	return http(
```

Comment:
no reason to change this. You can still do `export const setType = action => http()`. #nitpicking